src/Symfony/Component/Mailer/Bridge/Azure/Transport/AzureApiTransport.php

```diff
@@ -160,7 +160,7 @@ private function getMessageAttachments(Email $email): array
 
             $att = [
                 'name' => $filename,
-                'contentInBase64' => base64_encode(str_replace("\r\n", '', $attachment->bodyToString())),
+                'contentInBase64' => base64_encode(str_replace("\r\n", '', $attachment->getBody())),
```

Comment:
I'm doubtful about the str_replace, can you double-check?

Reply:
It doesn't make sense to me either, removed